tests/cql/CqlDateTimeOperatorsTest.xml

```diff
@@ -545,7 +546,11 @@
 	<group name="Duration">
 		<test name="DateTimeDurationBetweenYear">
 			<expression>years between DateTime(2005) and DateTime(2010)</expression>
-			<output>5</output>
+			<output>Interval[ 4, 5 ]</output>
+			<!-- TODO: How to handle the fact the question is resulting in an
+			undertainty interval and that CQL/ELM seem to provide no direct way of
+			selecting the same value, conceptually an implementation internal;
+			currently Equivalent() results in null from comparing with an Interval. -->
```

Comment:
This is actually an uncertainty between 4 and 5.  From [the CQL spec](http://build.fhir.org/ig/HL7/cql/05-languagesemantics.html#determining-difference-and-duration):
> The lower bound of that range is found by determining the duration between the maximum possible value of the first comparand and the minimum possible value of the second comparand; and the upper bound is determined using the minimum possible value of the first comparand and the maximum possible value of the second.

Using that definition, we have the following lower and upper values:
* lower: (whole) years between 2005-12-31T23:59:59.99 and 2010-01-01T00:00:00.0 = `4`
* upper: (whole) years between 2005-01-01T00:00:00.0 and 2010-12-31T23:59:59.999 = `5`